app/views/events/_form.html.erb

```diff
@@ -45,7 +45,7 @@
     <%= f.label :website, 'Website' %>
   </p>
   <p>
-    <%= f.text_field :website, :placeholder => "i.e. myconference.com" %>
+    <%= f.text_field :website, :placeholder => "i.e. https://myconference.com" %>
```

Comment:
I would have left this the way it was (less visual clutter). But perhaps "e.g." (for example given, or some Latin equivalent) instead of "i.e." (for Latin id est -> "it is")